Makefile

```diff
@@ -408,6 +408,10 @@ bin/.bootstrap: $(GITHOOKS) vendor/modules.txt | bin/.submodules-initialized
 
 IGNORE_GOVERS :=
 
+
+ifneq ($(TC_BUILD_BRANCH),)
+is-ci := 1
```

Comment:
Nit: would be good to have a way to opt out of this behavior, like `COCKROACH_MAKE_SKIP_GAZELLE` or similar. Otherwise the only way to opt out would be to set `TC_BUILD_BRANCH` on your dev machine which probably has other consequences you don't want.

Reply:
> a way to opt out
like, say, `bazel build //pkg/cmd/cockroach`?